hack/images/merger/merger.sh

```diff
@@ -3,10 +3,16 @@
 SRC=${SRC:-"/yamls"}
 OUT=${OUT:-"/merged.yaml"}
 
-# prefix each file with its filename
 for filename in "${SRC}"/*; do
   filename=$(basename -- "$filename")
   prefix="${filename%.*}"
+
+  # remove value key if exists
```

Comment:
For consistency reason, it would be good to also have `unpackValue` conf option here. When I went through our manifests, it was hard to understand why it works in this way, as this is a "hidden" functionality.

But let's do that on a new follow-up PR. It's not a blocker for merging this one 👍